ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -133,50 +137,173 @@ export const AdminUsers = withUserProfile()(class extends React.Component<
     return new Date(date).toString().split(' ').slice(1, 5).join(' ');
   }
 
-  convertProfilesToFields(profiles: Profile[]) {
-    return profiles.map(p => ({
-      ...p,
-      name: <a onClick={() => navigate(['admin', 'users', usernameWithoutDomain(p.username)])}>
-        {p.familyName + ', ' + p.givenName}
+  getAccessModuleString(user: AdminTableUser, key: string) {
+    const completionTime = user[key + 'CompletionTime'];
+    const bypassTime = user[key + 'BypassTime'];
+    if (completionTime) {
+      return '✔';
+    } else if (bypassTime) {
+      return 'B';
+    } else {
+      return '';
+    }
+  }
+
+  convertProfilesToFields(users: AdminTableUser[]) {
+    return users.map(user => ({
+      audit: <a
+        href={`/admin/user-audit/${usernameWithoutDomain(user.username)}`}
+        target='_blank'>
+        link
+      </a>,
+      bypass: <AdminUserBypass user={user}/>,
+      complianceTraining: this.getAccessModuleString(user, 'complianceTraining'),
+      contactEmail: user.contactEmail,
+      dataUseAgreement: this.getAccessModuleString(user, 'dataUseAgreement'),
+      eraCommons: this.getAccessModuleString(user, 'eraCommons'),
+      firstRegistrationCompletionTime: moment.unix(user.firstRegistrationCompletionTime / 1000).format('lll'),
+      firstRegistrationCompletionTimestamp: user.firstRegistrationCompletionTime,
+      firstSignInTime: moment.unix(user.firstSignInTime / 1000).format('lll'),
+      firstSignInTimetsamp: user.firstSignInTime,
+      institutionName: user.institutionName,
+      name: <a
+        href={`/admin/users/${usernameWithoutDomain(user.username)}`}
+        target='_blank'>
+        {user.familyName + ', ' + user.givenName}
       </a>,
-      betaAccessRequestTime: this.convertDate(p.betaAccessRequestTime),
-      bypass: <AdminUserBypass profile={p}/>, disabled: p.disabled.toString(),
-      userLockout: <LockoutButton disabled={this.state.reloadingProfile === p}
-        profileDisabled={p.disabled}
-        onClick={() => this.updateUserDisabledStatus(!p.disabled, p)}/>,
-      audit: <a href={`/admin/user-audit/${usernameWithoutDomain(p.username)}`}>🕵️</a>
+      nameText: user.familyName + ' ' + user.givenName,
+      status: user.disabled ? 'Disabled' : 'Active',
```

Comment:
nit: disabled/enabled and inactive/active are kind of separate concepts (or at least Active is overloaded to mean "has been active in the app recently".